examples/viewer.py

```diff
@@ -733,6 +783,51 @@ def exit_event(self, event: Application.ExitEvent):
         event.accepted = True
         exit(0)
 
+    def draw_text(self, sensor_subtype):
+        self._shader.bind_vector_texture(self._cache.texture)
+        self._shader.transformation_projection_matrix = (
+            self._transformation_projection_window_text
+        )
+        self._shader.color = [1.0, 1.0, 1.0]
+        self._shader.outline_color = [1.0, 1.0, 1.0]
+        self._shader.outline_range = (0.45, 0.40)
+
+        sensor_subtype_string = self.get_sensor_subtype_string(sensor_subtype)
+        if self.mouse_interaction == MouseMode.LOOK:
+            mouse_mode_string = "LOOK"
+        elif self.mouse_interaction == MouseMode.GRAB:
+            mouse_mode_string = "GRAB"
+        placeholder = "Oprah"
+        self._window_text.render(
+            f"""
+{self.fps} FPS
+{placeholder} drawables, {placeholder} culled
+{sensor_subtype_string}
+Last {self.num_frames_to_track} frames:
+    Frame time: {placeholder} ms
+    CPU duration: {placeholder} ms
+    GPU duration: {placeholder} µs
+    Vertex fetch ratio: {placeholder}
+    Primitives clipped: {placeholder} %
+Mouse Interaction Mode: {mouse_mode_string}
+            """
+        )
+        self._shader.draw(self._window_text.mesh)
+
+    def get_sensor_subtype_string(self, sensor_subtype):
+        if sensor_subtype == SensorSubType.PINHOLE:
```

Comment:
It's not quite as pretty, but can't we just figure out a way to stringify the name of the enum? Makes it easier to extend. Something like
```python
    return str(sensor_subtype.value)
```

Reply:
If that is possible in Python, that would be sweet. I'll look into it. I just remember trying to do that in C++ one time and it was tough